geomesa-core/src/main/scala/org/locationtech/geomesa/core/index/Strategy.scala

```diff
@@ -118,17 +117,16 @@ trait Strategy {
                      schema: String,
                      featureEncoding: FeatureEncoding,
                      query: Query): Option[IteratorSetting] = {
-    if (iteratorConfig.useSFFI) {
-      Some(configureSimpleFeatureFilteringIterator(featureType, ecql, schema, featureEncoding, query))
-    } else None
+    if (iteratorConfig.useSFFI) Some(configureSimpleFeatureFilteringIterator(featureType, ecql, schema, featureEncoding, query))
```

Comment:
just convention... we're not super picky about it, but since it looks like the actual code didn't change I would roll it back.